.github/workflows/cclf-import-apply.yml

```
@@ -41,4 +41,13 @@ jobs:
           aws-region: ${{ vars.AWS_REGION }}
       - run: terraform init -reconfigure -backend-config=../../backends/$TF_VAR_app-$TF_VAR_env.s3.tfbackend
       - run: terraform apply -auto-approve
+      - name: notify slack
+        id: slack
+        uses: slackapi/slack-github-action@v1.26.0
+        if: ${{ failure() }}
+        with:
+          channel-id: 'C04UG13JF9B'
+          slack-message: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

Should say that the run failed:

```suggestion
          slack-message: "Terraform apply failure: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

---

And sorry, I can see the miscommunication in my previous comment. I mean "only include a link to the failing run" in that no other links need to be included.